src/sentry/conf/server.py

```diff
@@ -881,7 +881,7 @@ def create_partitioned_queues(name):
     # Enable functionality for rate-limiting events on projects.
     "projects:rate-limits": True,
     # Enable functionality for sampling of events on projects.
-    "projects:sample-events": True,
+    "projects:sample-events": False,
```

Comment:
Do we need to retain this feature switch if its not used on saas?

Reply:
It's still being referenced right now, will remove it together with the code